src/Components/Table.js

```diff
@@ -0,0 +1,34 @@
+import { projects } from "../constants/projects";
```

Comment:
Thanks a lot for refactoring this portion. 🚀

It would make more sense to add the constants in Projects.js and send it as a prop to Table component.
This would help us replace the mockData with the actual API response in future effortlessly.

Reply:
Done.